com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
+
+  "app-id": "com.openmusicplayer.OMP",
+
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+
+  "sdk": "org.gnome.Sdk",
+
+
+  "finish-args":
+  [
+
+    "--socket=x11",
+    "--share=network",
+    "--filesystem=host",
+    "--socket=pulseaudio"
+
+  ],
+
+  "build-options" :
+  { 
+
+    "cxxflags": "-g -std=c++17 -lstdc++fs"
+
+   },
+
+  "command": "omp",
+
+  "modules":
+  [
+
+    {
+
+      "name": "taglib",
+      "buildsystem": "cmake-ninja",
+      "config-opts": ["-DENABLE_TESTING=OFF" , "-DBUILD_SHARED_LIBS=ON", "-DCMAKE_INSTALL_PREFIX:PATH=/app"],
+      "sources": [ { "type": "git", "url": "https://github.com/taglib/taglib.git", "branch": "v1.11.1" } ]
+
+    },
+
+
+
+    {
+
+      "name": "mm-common",
+      "cleanup": [ "/" ],
```

Comment:
What is it supposed to do?

Reply:
Are you asking about the cleanup option or mm-common?

Reply:
If you want to clear everything for a given module, use this:
```
      "cleanup": [ "*" ],
```

This can be useful for header files or for tools that are only needed at build time.